libshortfin/CMakeLists.txt

```diff
@@ -46,9 +39,77 @@ list(APPEND CMAKE_MODULE_PATH
 include(shortfin_library)
 
 # Dependencies.
-find_package(spdlog)
-find_package(xtensor)
-find_package(IREERuntime)
+
+## fmt
```

Comment:
We need to condition this on some kind of SHORTFIN_BUNDLE_DEPS option and if it is false, use find_package. Not bundling the deps is important for certain scenarios and package managers.

Reply:
done